Readme.md

```diff
@@ -3,21 +3,43 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E55)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* **[Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))**
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# How to show the ASPxPopupControl
+# Popup Control for ASP.NET Web Forms - How to show the Popup Control
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e55/)**
 <!-- run online end -->
+This example demonstrates how to show a pop-up window.
+
+## Overview
+
+Handle a client-side or server-side event and call one of the following methods to show the [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl)'s pop-up window:
```

Comment:
`client-side or server-side event` Any?

Reply:
I've rewrite the Readme's text. Could you please review it again?